### PR TITLE
ZCS-13650 : Admin Console functionality to add translation for newly added labels for rule disallowing username in the password

### PR DESCRIPTION
**Pull request by kaishavsaini**
Added Translation for NAD_AllowUsernameInPassword and ERROR_PASSWORD_CONTAIN_USERNAME

### CODE REVIEW

File: WebRoot/messages/ZaMsg_es.properties [hunk 1]
```diff
@@ -1182,6 +1182,7 @@ NAD_Remove = Eliminar
 NAD_ForwardTo = Reenviar correo a
 NAD_COSName = Nombre de Clase de servicio
 NAD_PwdLocked = Evitar que el usuario cambie la contrase\u00f1a
+NAD_AllowUsernameInPassword = Permitir nombre de usuario en contraseña
```
Comment (sakshirawat0247) [inline WebRoot/messages/ZaMsg_es.properties:1185]: replace ñ with \u00f1

Comment (kaishavsaini) [reply WebRoot/messages/ZaMsg_es.properties]: done

**Review by sakshirawat0247 — APPROVED** [review]

File: WebRoot/messages/ZaMsg_ja.properties [hunk 2]
```diff
@@ -1178,6 +1178,7 @@ NAD_Remove = \u524a\u9664
 NAD_ForwardTo = \u30e1\u30fc\u30eb\u306e\u8ee2\u9001\u5148
 NAD_COSName = \u63d0\u4f9b\u30b5\u30fc\u30d3\u30b9\u540d
 NAD_PwdLocked = \u30e6\u30fc\u30b6\u30fc\u304c\u30d1\u30b9\u30ef\u30fc\u30c9\u3092\u5909\u66f4\u3057\u306a\u3044\u3088\u3046\u306b\u3059\u308b
+NAD_AllowUsernameInPassword = \u30d1\u30b9\u30ef\u30fc\u30c9\u306b\u30e6\u30fc\u30b6\u30fc\u540d\u3092\u8a31\u53ef\u3059\u308b
```
Comment (k-kato) [inline WebRoot/messages/ZaMsg_ja.properties:1181]:
```suggestion
NAD_AllowUsernameInPassword = \u30d1\u30b9\u30ef\u30fc\u30c9\u306b\u30e6\u30fc\u30b6\u30fc\u540d\u3092\u542b\u3081\u308b\u3053\u3068\u3092\u8a31\u53ef
```

Comment (kaishavsaini) [reply WebRoot/messages/ZaMsg_ja.properties]: Done

**Review by k-kato — APPROVED** [review]

**Review by silentsakky — APPROVED** [review]

**Pull request merged**